### PR TITLE
Updated rule 2.4 with more clarification

### PR DESCRIPTION
**Pull request by DasCodes**
Divided the rule into subcategories for better understanding for the players.

### CODE REVIEW

File: rules.html [hunk 1]
```diff
@@ -122,10 +122,26 @@ <h2>2. Behavior and chat</h2>
             </li>
             <li>
                 <b>2.4 Toxicity.</b><br/>
-                Toxicity is not allowed on this server. Toxicity in chat can be for instance using racial slurs, sexual harassment or severely explicit language.
-                It can also be other non-chat actions which drive players away from the community, for instance portal trapping, luring newly joined players to your town so you can kill them or lock them in your town jail.<br/>
+                Toxicity is not allowed on this server. Toxicity in chat includes but is not limited to:<br/>
+                <ul>
```
Comment (KarlOfDuty) [inline rules.html]: I don't think there should be an inner list here. The outer list only has two nodes, the inner list and one extra list item added on at the end, doesn't really make much sense structurally.

**Comment by KarlOfDuty** [comment]
Actually I'll make that a file comment instead of a line comment.

File: rules.html [hunk 2]
```diff
@@ -122,10 +122,26 @@ <h2>2. Behavior and chat</h2>
             </li>
             <li>
                 <b>2.4 Toxicity.</b><br/>
-                Toxicity is not allowed on this server. Toxicity in chat can be for instance using racial slurs, sexual harassment or severely explicit language.
-                It can also be other non-chat actions which drive players away from the community, for instance portal trapping, luring newly joined players to your town so you can kill them or lock them in your town jail.<br/>
+                Toxicity is not allowed on this server. Toxicity in chat includes but is not limited to:<br/>
+                <ul>
+                    <li><b>2.4.a Harassment:</b> This includes but not limited to:</li>
+                    <li>
+                        <ul class="sub_list">
```
Comment (KarlOfDuty) [inline rules.html]: I don't think there should be an inner list here. The outer list only has two nodes, the inner list and one extra list item added on at the end, doesn't really make much sense structurally. They should be moved out to the outer list.

**Review by KarlOfDuty — CHANGES_REQUESTED** [review]

**Comment by KarlOfDuty** [comment]
Or apparently not, this will have to do than.

**Review by DasCodes — COMMENTED** [review]
It's a nested list. I was going to put just one list inside into the main list element first, but then made 2 nested list element inside the main list element.

The first nested list is the main element hosting the sub-rules, as it has two sections `2.4.a` and `2.4.b`.
```suggestion
                <ul>
                    <li><b>2.4.a Harassment:</b> This includes but not limited to:</li>
```
The second nested list element is for the individual lines for the first nested list element. I went with this nested design thinking if in future we will need to make more sub-rules like this, you can do it easily, organized way.

**Comment by DasCodes** [comment]
Actually, have to tweak the CSS a bit more to have a better alignment view in smaller screens.

File: rules.html (hunk 2 above)
Comment (DasCodes) [reply rules.html]: Test!

Comment (DasCodes) [reply rules.html]: Okay, wasn't letting me comment here before.

This is the same design you go with if you have multiple sub-menus in a dropdown navigation menu, and this is a more organized way than putting `<br/>` after every line.

Comment (DasCodes) [reply rules.html]: As there are two sub-lists under `2.4`, which are 
As both `2.4.a` and `2.4.b` as lists, they are both in their list under the main list `2.4`. And as `2.4.a` have a list of itself too, it has another list for that.

**Comment by KarlOfDuty** [comment]
Yes, that is what I'm saying, just merge them into one list. The second section is barely longer than the other list entries.

**Comment by DasCodes** [comment]
You mean the list inside `2.4.a` to be merged with its parent list?

**Comment by KarlOfDuty** [comment]
Yes. There is no need for two sections there.

**Comment by DasCodes** [comment]
Well, it's possible yes, but what I am saying is that you'll need to make a new line for each one of those lists, plus will need to increase the line-height for that to make it look clear, which might also break other CSS changes as the original CSS has only bare minimum changes.

**Review by KarlOfDuty — APPROVED** [review]

**Comment by DasCodes** [comment]
Should probably delete the other branches, for now. I have not been able to get to design a newer version of the site.

**Pull request merged**